dvc/scm/git/__init__.py

```diff
@@ -235,3 +235,52 @@ def belongs_to_scm(self, path):
 
     def get_tree(self, rev):
         return GitTree(self.git, rev)
+
+    def _get_diff_trees(self, a_ref, **kwargs):
+
+        from gitdb.exc import BadObject, BadName
+
+        trees = {"a_tree": None, "b_tree": None}
+        commits = []
+        try:
+            if "b_ref" in kwargs and kwargs["b_ref"] is not None:
+                b_ref = kwargs["b_ref"]
+                a_commit = self.git.commit(a_ref)
+                b_commit = self.git.commit(b_ref)
+                commits.append(str(a_commit))
+                commits.append(str(b_commit))
+                # other way of getting the diff between commits :
+                # self.git.diff(a_commit, b_commit) returns the diff string
+                # by reading the files in a, b commits and computing the diff
+            else:
+                a_commit = self.git.head.commit
+                b_commit = self.git.commit(a_ref)
```

Comment:
b_commit = ... a_ref looks pretty strange :) That's why I would probably just name params just `--ref`.